docs/custom-model-integration/Dockerfile.reference

```diff
@@ -0,0 +1,39 @@
+FROM python:3.10-slim@sha256:684b1aaf96a7942b3c3af438d162e0baa3510aa7af25ad76d238e0c746bdec79
```

Comment:
This file is how I built the reference image `ghcr.io/azure/kaito/llm-reference-preset:latest`. 

I can remove from PR, deemed not necessary

Reply:
I like it, we should keep it.

Reply:
But please add the cmd line to build this image in the Makefile.

Reply:
added